simulation/agents.py

```diff
@@ -513,21 +513,24 @@ def get_total_heating_system_costs(
 
         return unit_and_install_costs, fuel_costs_net_present_value, -subsidies
 
-    def reset_heating_system_hassle(self, heating_system_hassle_factor: float):
-        # Ensure landlords have a hassle factor of at least 0.4
-        if heating_system_hassle_factor < 0.4:
-            if (
-                self.occupant_type == OccupantType.RENTED_PRIVATE
-                or self.occupant_type == OccupantType.RENTED_SOCIAL
-            ):
-                return 0.4
-            else:
-                return heating_system_hassle_factor
+    def reset_heating_system_hassle(
```

Comment:
Previously you have a hard floor of this value to be minimum 0.4 for renters, but you've removed this hard floor. If via CLI someone changes the input to 0.3 for example, this will be allowed.

Is the removal of the hard floor intended?

Reply:
Yes, originally I implemented a hard floor on the `heating_system_hassle_factor` for rented households, but then decided it would be better to have this as an input argument instead. The user can choose nay value between 0 and 1 now (not limited to >0.4).